xbmc/addons/AddonSystemSettings.h

```diff
@@ -21,6 +21,12 @@ const int AUTO_UPDATES_ON = 0;
 const int AUTO_UPDATES_NOTIFY = 1;
 const int AUTO_UPDATES_NEVER = 2;
 
+enum class AddonTrustedUpdateMode : int
```

Comment:
```suggestion
enum class AddonTrustedUpdateMode
```
`int` is not necessary